src/jit/gentree.cpp

```diff
@@ -8446,11 +8446,17 @@ GenTree** GenTreeUseEdgeIterator::GetNextUseEdge() const
             }
 
         // Call, phi, and SIMD nodes are handled by MoveNext{Call,Phi,SIMD}UseEdge, repsectively.
+        //
+        // If FEATURE_MULTIREG_ARGS is enabled, so PUTARG_STK nodes also have special handling.
         case GT_CALL:
         case GT_PHI:
 #ifdef FEATURE_SIMD
         case GT_SIMD:
 #endif
+#if FEATURE_MULTIREG_ARGS
+        case GT_PUTARG_STK:
+#endif
+
```

Comment:
All of these break statements lead to  the
unreached() call outside of the switch.
If it is not expected to get to the unreached() then shouldn't we have an assert before the break